lib/travis/build/script/node_js.rb

```diff
@@ -2,10 +2,6 @@ module Travis
   module Build
     class Script
       class NodeJs < Script
-        DEFAULTS = {
```

Comment:
What is supposed to happen if `.nvmrc` does not exist and no value is set in `node_js` in `.travis.yml`?